Page crashes when error message contains braces

When I tried to create the index, the page crashed with the following stacktrace being shown:

```
java.lang.IllegalArgumentException: can't parse argument number: "error":{"root_cause":[{"type":"class_cast_exception"
	at java.base/java.text.MessageFormat.makeFormat(MessageFormat.java:1451)
	at java.base/java.text.MessageFormat.applyPattern(MessageFormat.java:491)
	at java.base/java.text.MessageFormat.<init>(MessageFormat.java:370)
	at java.base/java.text.MessageFormat.format(MessageFormat.java:859)
	at org.kitodo.production.helper.Helper.getTranslation(Helper.java:443)
	at org.kitodo.production.helper.Helper.setMessage(Helper.java:297)
	at org.kitodo.production.helper.Helper.setMessage(Helper.java:289)
	at org.kitodo.production.helper.Helper.setErrorMessage(Helper.java:73)
	at org.kitodo.production.helper.Helper.setErrorMessage(Helper.java:160)
	at org.kitodo.production.forms.IndexingForm.createMapping(IndexingForm.java:220)
	…
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.NumberFormatException: For input string: ""error":{"root_cause":[{"type":"class_cast_exception""
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
	at java.base/java.lang.Integer.parseInt(Integer.java:638)
	at java.base/java.lang.Integer.parseInt(Integer.java:770)
	at java.base/java.text.MessageFormat.makeFormat(MessageFormat.java:1449)
	... 103 more
```

The reason for the error itself was an outdated ElasticSearch version, but, no error bar is showing, but the error page, because the Formatter tries to parse the JSON array in the answer.

**Goal:** Handle fallback if message formatter fails, and show error message.

--- a/Kitodo/src/main/java/org/kitodo/production/helper/Helper.java
+++ b/Kitodo/src/main/java/org/kitodo/production/helper/Helper.java
@@ -442,7 +442,12 @@ public class Helper {
      */
     public static String getTranslation(String title, String... insertions) {
         String pattern = getString(desiredLanguage(), title);
-        String message = MessageFormat.format(pattern, (Object[]) insertions);
+        String message = pattern;
+        try {
+            message = MessageFormat.format(pattern, (Object[]) insertions);
+        } catch (IllegalArgumentException e) {
+            logger.catching(Level.WARN, e);
+        }
         return appendUnusedInsertions(message, insertions);
     }
 

